tests/Unit/Evolution/Particles/MonteCarlo/Test_TakeTimeStep.cpp

```diff
@@ -22,7 +22,6 @@ namespace{
 
 void test_flat_space_time_step(const bool skip) {
   CHECK(true);
-  // This test FPEs because the ghost zone handling is broken.
   if (skip) {
     return;
   }
```

Comment:
Maybe just
```cpp
void test_flat_space_time_step() {
```
since presumably we won't need to skip this test anymore? :D

Reply:
Done (rebased and merged)